packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -342,36 +357,48 @@ export default class PciInstancesAddController {
     });
   }
 
-  getFilteredRegions() {
-    this.availableRegions = {};
-    this.unavailableRegions = {};
-
-    const planCode = `${this.model.flavorGroup.name}.consumption`;
-
+  fetchInstancesAvailability() {
+    this.isInstancesAvailabilityFetching = true;
     return this.PciProjectsProjectInstanceService.getProductAvailability(
       this.projectId,
       this.coreConfig.getUser().ovhSubsidiary,
+      'instance',
     ).then((productCapabilities) => {
-      // New plancodes has been introduced for localzones which will have region names after plan codes names like ***.consumption.EU-WEST-LZ-BRU-A
-      // So we need to consider all the plancodes which starts with "**flavorName**.consumption"
-      const productCapability = productCapabilities.plans?.filter((plan) =>
-        plan.code?.startsWith(planCode),
-      );
+      this.instancesProductCapabilities = productCapabilities;
+      this.isInstancesAvailabilityFetching = false;
```

Comment:
Idem here, I guess it misses a finally statement or at least a catch one to prevent from infinite loading ?

Reply:
Same here, errors were never handled